### ISSUE TITLE
Missing `&` in alignify

### ISSUE THREAD

**Issue by polyluxus** [GaurangTandon/MarkdownShortcutsForSE#20]
Input:
```
M(\ce{H2SO4}) = 98.1 g/mol
M(\ce{H2O})   = 18.0 g/mol
\rho(\ce{H2SO4}) = 98.1 g/L
c(\ce{H2SO4})= \pu{1 mol/L
M_{\mathrm{contr}} \times V_{\mathrm{contr}} =
 M_{\mathrm{dil}}\times V_{\mathrm{dil}}
\Longrightarrow V_{\mathrm{dil}} =
\frac{M_{\mathrm{conctr}}\times V_{\mathrm{conctr}}}{M_{\mathrm{dil}}}
```
Output:
```
\begin{align}
M(\ce{H2SO4}) = 98.1 g/mol\\
M(\ce{H2O})   = 18.0 g/mol\\
\rho(\ce{H2SO4}) = 98.1 g/L\\
c(\ce{H2SO4})= \pu{1 mol/L\\
M_{\mathrm{contr}} \times V_{\mathrm{contr}} &=
M_{\mathrm{dil}}\times V_{\mathrm{dil}}\\
\Longrightarrow V_{\mathrm{dil}} &=
\frac{M_{\mathrm{conctr}}\times V_{\mathrm{conctr}}}{M_{\mathrm{dil}}}
\end{align}
```

Only the lines ending on `=` get the ampersand.

Ref: [Determining volume of a sulfuric acid solution with a density of 4.9 g/l](https://chemistry.stackexchange.com/q/102427/4945)

**Comment by GaurangTandon** [comment]
Fixed in v1.8.0

Test case:

```
A = B
C=D

\ce{CH3=O} -> \ce{H2O}
\ce{CH3=O = H2O}

M(\ce{H2SO4}) = 98.1 g/mol
M(\ce{H2O})   = 18.0 g/mol
\rho(\ce{H2SO4}) = 98.1 g/L
c(\ce{H2SO4})= \pu{1 mol/L
M_{\mathrm{contr}} \times V_{\mathrm{contr}} =
 M_{\mathrm{dil}}\times V_{\mathrm{dil}}
\Longrightarrow V_{\mathrm{dil}} =
\frac{M_{\mathrm{conctr}}\times V_{\mathrm{conctr}}}{M_{\mathrm{dil}}}
```

**Comment by polyluxus** [comment]
Nicely done, thank you. Just a note: In my input and subsequently yours, there was a brace missing, so if you apply it, it won't render. For the sake of testing:
```
A = B
C=D

\ce{CH3=O} -> \ce{H2O}
\ce{CH3=O = H2O}

M(\ce{H2SO4}) = \pu{98.1 g/mol}
M(\ce{H2O})   = \pu{18.0 g/mol}
\rho(\ce{H2SO4}) = \pu{98.1 g/L}
c(\ce{H2SO4})= \pu{1 mol/L}
M_{\mathrm{contr}} \times V_{\mathrm{contr}} =
 M_{\mathrm{dil}}\times V_{\mathrm{dil}}
\Longrightarrow V_{\mathrm{dil}} =
\frac{M_{\mathrm{conctr}}\times V_{\mathrm{conctr}}}{M_{\mathrm{dil}}}
```
(Also there were some constructs that would need `\pu{}`, so I added that, too.)
